### PR TITLE
neovide: Add version 0.5.0

### PR DESCRIPTION
**Pull request by boxdox**
No Nonsense Neovim Client in Rust

https://github.com/Kethku/neovide

### CODE REVIEW

File: bucket/neovide.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "url": "https://github.com/Kethku/neovide/releases/download/0.5.0/neovide.exe",
+  "hash": "FA8453D74B6833E7912C049FAA9880544CB629E3E610F1717639C8198A26CD85",
+  "bin": "neovide.exe",
+  "checkver": {
+    "github": "https://github.com/Kethku/neovide"
+  },
+  "autoupdate": {
+    "url": "https://github.com/Kethku/neovide/releases/download/$version/neovide.exe"
+  }
```
Comment (Ash258) [inline bucket/neovide.json]:
```suggestion
{
    "version": "0.5.0",
    "description": "A simple GUI for Neovim",
    "homepage": "https://github.com/Kethku/neovide",
    "license": "MIT",
    "suggest": {
        "neovim": "neovim"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/Kethku/neovide/releases/download/0.5.0/neovide.exe",
            "hash": "fa8453d74b6833e7912c049faa9880544cb629e3e610f1717639c8198a26cd85"
        }
    },
    "bin": "neovide.exe",
    "shortcuts": [
        [
            "neovide.exe",
            "Neovide"
        ]
    ],
    "checkver": "github",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/Kethku/neovide/releases/download/$version/neovide.exe"
            }
        }
    }
}
```

Comment (boxdox) [reply bucket/neovide.json]: Hey, sorry this was my first app manifest and I didn't realize this neovide needed neovim.
I'll update it.

Doesn't `checkver` require github url?

Comment (Ash258) [reply bucket/neovide.json]: https://scoop-docs.now.sh/docs/concepts/App-Manifest-Autoupdate.html#adding-checkver-to-a-manifest

Comment (Ash258) [reply bucket/neovide.json]: https://scoop-docs.now.sh/docs/concepts/App-Manifest-Autoupdate.html#adding-checkver-to-a-manifest

File: bucket/neovide.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "neovim": "neovim"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Kethku/neovide/releases/download/0.5.0/neovide.exe",
+            "hash": "fa8453d74b6833e7912c049faa9880544cb629e3e610f1717639c8198a26cd85"
+        }
+    },
+    "bin": "neovide.exe",
+    "shortcuts": [["neovide.exe", "Neovide"]],
```
Comment (Ash258) [inline bucket/neovide.json]:
```suggestion
    "shortcuts": [
        [
            "neovide.exe",
            "Neovide"
        ]
    ],
```

File: bucket/neovide.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "version": "0.5.0",
+    "homepage": "https://github.com/Kethku/neovide",
+    "description": "No Nonsense Neovim Client in Rust",
```
Comment (Ash258) [inline bucket/neovide.json]:
```suggestion
    "description": "A simple GUI for Neovim",
    "homepage": "https://github.com/Kethku/neovide",
```

**Comment by Ash258** [comment]
Is there any special dependency?

I just do not work.

image

No window or anything is spawned after execution.

**Comment by boxdox** [comment]
It works perfectly fine in my PC, from both cmd and start menu.
Capture

[Documentation suggests that](https://github.com/Kethku/neovide#install):
>Installing should be as simple as downloading the binary, making sure nvim.exe with version 0.4 or greater is on your path, and running it. Everything should be self contained.

**Comment by Ash258** [comment]
image

**Comment by boxdox** [comment]
Can you try to download the binary from github directly and check ?

**Comment by boxdox** [comment]
@ash258 any updates?

**Comment by Ash258** [comment]
I am not going to waste more time on this. It just do not work for me and not going to do anything about it. If someone else confirm it work, I might merge this, but I doubt it.)

**Comment by TheRandomLabs** [comment]
This works fine for me.

**Comment by JochemKlingeler** [comment]
Works for me as well.

It lists neovim as a `suggest`ed install, but should it not be marked as a `depends` instead?
Neovide needs Neovim to be installed for it to work.

**Comment by TheRandomLabs** [comment]
`versions/neovim-nightly` exists. This is where the `provides` property or similar would be useful.

**Comment by MunsterPlop** [comment]
I just tested the recipe using `"depends": "neovim"` and it works just fine.

File: bucket/neovide.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.5.0",
+    "description": "A simple GUI for Neovim",
+    "homepage": "https://github.com/Kethku/neovide",
+    "license": "MIT",
+    "suggest": {
+        "neovim": "neovim"
+    },
```
Comment (MunsterPlop) [inline bucket/neovide.json:8]:
```suggestion
"depends": "neovim",
```

Comment (Ash258) [reply bucket/neovide.json:8]: No.

Depends is not used in these cases.

Comment (MunsterPlop) [reply bucket/neovide.json:8]: Alright. Fair enough.
Is there any chance of this PR being merged then?
A few people already confirmed that it works as long as you satisfy the neovim dependency.

**Pull request merged**